src/main/java/io/swagger/swaggerhub/plugin/SwaggerHubUpload.java

```diff
@@ -70,14 +103,39 @@ public void execute() throws MojoExecutionException {
                 + ", isPrivate: " + isPrivate
                 + ", definitionDirectory: " + definitionDirectory
                 + ", definitionFileNameRegex: " + definitionFileNameRegex
-                + ", uploadType: " + uploadType);
+                + ", uploadType: " + uploadType
+                + ", scmProvider: " + scmProvider
+                + ", scmToken: " + (null!=scmToken ? scmToken.replaceAll(".", "*"):"")
```

Comment:
formatting. Also it can be useful to log either the last couple of characters or first and last to help the user confirm what token is being sent without compromising it